### PR TITLE
Add sentencepiece package

### PR DESCRIPTION
**Pull request by justinvreeland**
Builds 4 packages 1 for each python3 version and the libraries.

New sentencepiece package

Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [X] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [X] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

**Comment by justinvreeland** [comment]
@pnasrat

File: sentencepiece.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package]} installed for python${{range.key}}
+    pipeline:
+      - runs: python${{range.key}} -m pip install ./python "--root=${{targets.subpkgdir}}" --prefix=/usr
+      - uses: strip
+    dependencies:
+      runtime:
+        - py3-${{vars.pypi-package}}
```
Comment (pnasrat) [inline sentencepiece.yaml:52]: the `py3-${{vars.pypi-package}}` dependency looks incorrect

The py3-sentencepiece virtual should be handled by the provides below. You allso need to use provider-priority set to `${{range.value}}` for these subpackages.

**Review by pnasrat — DISMISSED** [review]

**Review by smoser — CHANGES_REQUESTED** [review]
My only question here is why you named this 'sentancepiece' rather than 'py3-sentancepiece'.  setuptools is named 'py3-setuptools' but the "upstream name" (and name in pypi) is "setuptools".

Why differ here?

I'm only discussing consistency, not which is right or wrong.  we have ~ 525 py-* packages.

**Comment by justinvreeland** [comment]
> My only question here is why you named this 'sentancepiece' rather than 'py3-sentancepiece'. setuptools is named 'py3-setuptools' but the "upstream name" (and name in pypi) is "setuptools".
> 
> Why differ here?

This was the first one I put up and I didn't really know how we were naming things. It should be named to match the others.

**Review by smoser — APPROVED** [review]

**Pull request merged**